grpc_check/tests/fixtures/ca-config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "signing": {
+    "profiles": {
+      "default": {
+        "usages": ["signing", "key encipherment", "server auth", "client auth"],
+        "expiry": "8760h"
```

Comment:
Is it possible to increase this value so we won't have to update the certificates next year?

Reply:
Increased to `876600h`(100 years).